README.md

```diff
@@ -55,3 +55,30 @@ $ dune exec -- examples/submit.exe --submission-service=capnp-secrets/submission
 ```
 
 You should then be able to watch the pipeline in action at `http://localhost:8080`.
+
+### A solver worker by docker-compose
+
+Create `~/local.data`, this directory is needed to store all the data and share all capnp-secrets files between the
```

Comment:
Other pipelines (ocaml-ci, opam-repo-ci and ocaml-docs-ci) create a `capnp-secrets` directory when running them locally. This should use the same convention.

Reply:
Here, you can find an exemple on ocaml-ci about how to ensure existance: https://github.com/ocurrent/ocaml-ci/blob/master/service/main.ml#L55 :wink: